db/migrate/201509151000_create_five_a_day_counts_table.rb

```diff
@@ -7,6 +7,6 @@ def change
       t.timestamps null: false
     end
 
-    add_index :five_a_day_counts, [:user_id, :total], unique: true
+    add_index :five_a_day_counts, [:user_id, :total]
```

Comment:
Let's have an index (not unique) on just the user, since that's what we're doing the lookup on.

Also, let's have a `date` field (not timestamp), which has a unique index on [user_id, day] or whatever we want to call it, to enforce one row per day.

I don't think we need an index on total, since we're not going to be doing lookups on that.